### ISSUE TITLE
Remove duplication in Units Essential and Namespace data by Removing all the patterns that equals to `Other`

### ISSUE THREAD

**Issue by younies** [unicode-org/icu4x#5170]
In the CLDR specification: https://unicode.org/reports/tr35/tr35-numbers.html#Contents,

In Units Formatting patterns, if the pattern does not exist, choose to `Other` instead. Therefore, any pattern equal to `Other` should be deleted and `Other` should be used as the default value.

**Comment by robertbastian** [comment]
The `other` pattern is actually always required, so it can have its own field instead of being in the `ZeroMap`. This will often make the map be empty.

**Comment by sffc** [comment]
Agreed, though I hope that my vague idea of a plural pattern type would resolve this, too.
